tests/v2_validation/cattlevalidationtest/core/test_cattle_upgrade.py

```diff
@@ -0,0 +1,393 @@
+from common_fixtures import *  # NOQA
+from test_services_lb_ssl_balancer import validate_lb_services_ssl
+
+UPGRADE_SUBDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                              'resources/upgrade')
+
+pre_upgrade_stack_name = os.environ.get('PRE_UPGRADE_STACK_NAME')
+post_upgrade_stack_name = os.environ.get('POST_UPGRADE_STACK_NAME')
+preportsuffixnum = os.environ.get('PRE_PORT_SUFFIX_NUM')
+postportsuffixnum = os.environ.get('POST_PORT_SUFFIX_NUM')
+lbversion = os.environ.get('LB_VERSION')
```

Comment:
@soumyalj , Instead of asking for LB_VERSION , we should be able to get the current LB version from lb.instance.image settings.